model/common/src/icon4py/model/common/interpolation/rbf_interpolation.py

```diff
@@ -339,6 +339,12 @@ def index_offset(f):
     z_rbfmat_np = data_alloc.as_numpy(z_rbfmat)
     rhs_np = [data_alloc.as_numpy(x) for x in rhs]
     for i in range(num_elements):
+        # convert repeated indices back into -1
```

Comment:
I don't know whether we should do that here, as it fixes a problem from the test input data (the connectivities from the serialization) or rather do it in the test input prior to the tests. It for sure needs some more context as why we do it here. Maybe @msimberg  has an opionion on this.

Reply:
"correct" is not really a term that I would use in that context without definition. Either `ICON` is correct, then the serialized data is correct, or the gridfile is `correct`: then  are the two gridfile generators even doing the same thing? ... In any case: _I_ would say: we should keep the invalid indices and deal with them explicitly as for the order I think we should not assume that they are contiguous, because there are some ordering conventions for local neighborhoods in ICON like "vertices in cells being order counterclockwise", these are not very transparent (to me) and I don't know whether ICON keeps them consistently by exactly doing things like moving missing indices to the end.